CHANGELOG.MD

```diff
@@ -1,3 +1,21 @@
+#### v2.1.0
+
+- performance testing (browserMob and TTFB timings)
```

Comment:
I would rather add something like that:
- `added a possibility to test performance with browserMob (save `.har` file and then compare TTFB values)` - something like that